apps/static-site/src/content/docs/1_guides/0_managing-manifests.mdx

```diff
@@ -139,7 +139,8 @@ Event Stores (Replication Targets):
 
 A ReJot workspace is a collection of manifests that are managed together. It's useful for local
 development use cases where you're typically only using a single event store and only want to run a
-single sync service.
+single sync service. It can also be used to combine for example secrets and other configuration
+across multiple manifests.
```

Comment:
add a note that datastores/connections/etc cannot be overridden, as no duplicates are allowed.